README.rst

```diff
@@ -3,6 +3,48 @@ django-pldp
 
 Reusable Django app that implements the structure implied by the PDLP.
 
+Setup
+-----
+
+Add necessary things to your installed apps:
+
+```
+INSTALLED_APPS = (
```

Comment:
High level comment!

I feel a little reluctant about breaking the PLDP specifications into different apps, mainly, because we have Foreign key relations among them, suggesting that these models belong together in a single app. [This sarticle](https://medium.com/@DoorDash/tips-for-building-high-quality-django-apps-at-scale-a5a25917b2b5) discusses why having too many apps can cause headaches (mainly, the section "Be careful about applications").

I could imagine a structure, wherein we have the pldp app, and the pldp app has a models directory. The models directory contains individual files for each category, something like:

```
├── pldp
│   ├── init.py
│   ├── management
│   │       └── commands
│   │           └── initialize_pldp.py
│   ├── migrations
│   ├── models
│   │   ├── agency.py
│   │   ├── location.py
│   │   ├── study.py
│   │   ├── survey.py
```

I really hate to suggest a refactor of the architecture, given that the tool's primary author had something different in mind, but I am having trouble making a good argument for these different apps. @beamalsky let's talk about this and potentially consult with another Django expert on our team.

Reply:
Thanks for the link! I don't see a reason why these need to be separate apps, rather than just separate model files.